src/main/java/wellnus/atomichabit/command/HelpCommand.java

```diff
@@ -17,14 +17,16 @@ public class HelpCommand extends Command {
     public static final String COMMAND_DESCRIPTION = "help - Get help on what commands can be used "
             + "in Atomic Habit WellNUS++";
     public static final String COMMAND_USAGE = "usage: help [command-to-check]";
-    private static final String BAD_COMMAND_MESSAGE = "help does not take in any arguments!";
+    private static final String BAD_COMMAND_MESSAGE = "Invalid arguments given, 'help' should not take in any "
+            + "arguments!!";
     private static final String COMMAND_KEYWORD = "help";
     private static final String NO_FEATURE_KEYWORD = "";
     private static final String HELP_PREAMBLE = "Input `help` to see all available commands.\n"
             + "Input `help [command-to-check] to get usage help for a specific command.\n"
             + "Here are all the commands available for you!";
-    private static final String ERROR_UNKNOWN_COMMAND = "Sorry, we couldn't find that command!\n"
-            + "To find a command accessible in this part of WellNUS++, try `help`!";
+    private static final String ERROR_UNKNOWN_COMMAND = "Invalid command issued!";
+    private static final String COMMAND_INVALID_COMMAND_NOTE = "Please try 'help' command to check the "
```

Comment:
For this particular command(`help`), I think you legit don't need any additional messages lol

Reply:
Unless this is replaced with the `COMMAND_USAGE`